cmd/tools/grafana/grafana.go

```diff
@@ -409,7 +432,7 @@ func importFiles(dir string, folder Folder) {
 		return
 	}
 
-	releaseVersion := strings.ReplaceAll(harvestRelease, ".", "-")
+	uidSuffix := strings.ReplaceAll(folder.name, ".", "-")
```

Comment:
This should be harvest release. 
Intension would be having unique uid across release to avoid overriding of previous dashboard.

Reply:
okay. I changed it to support below usecase.

When i run below 2 commands, abc folder is empty. Same uid means, second command below moves cmode dashboards from abc to default generatewd folder. I'll aslo add harvestversion to uid.

1. bin/grafana import --addr my.grafana.server:3000 --serverfolder abc --directory /grafana/dashboards/cmode
2. bin/grafana import --addr my.grafana.server:3000